README.rst

```diff
@@ -490,9 +490,9 @@ Python-Django
 Python-Pyramid
 ^^^^^^^^^^^^^^
 
-* `pyramid-cookiecutter-alchemy`_: A Cookiecutter (project template) for creating a Pyramid project using SQLite for persistent storage, SQLAlchemy for an ORM, URL dispatch for routing, and Jinja2 for templating.
-* `pyramid-cookiecutter-starter`_: A Cookiecutter (project template) for creating a Pyramid starter project using URL dispatch for routing and either Jinja2, Chameleon, or Mako for templating.
-* `pyramid-cookiecutter-zodb`_: A Cookiecutter (project template) for creating a Pyramid project using ZODB for persistent storage, traversal for routing, and Chameleon for templating.
+* `pyramid-cookiecutter-alchemy`_: [DEPRECATED - Please use `pyramid-cookiecutter-starter`_ instead] A Cookiecutter (project template) for creating a Pyramid project using SQLite for persistent storage, SQLAlchemy for an ORM, URL dispatch for routing, and Jinja2 for templating.
```

Comment:
Please drop deprecated information from cookiecutter readme. I checked your repos and you have this note there. Why do we need to keep this outdated links in this doc?

Reply:
Done!